test/cookies/manifest.json

```diff
@@ -0,0 +1,161 @@
+{
+    "http_server": {
+        "addr": ":9999",
+        "dir": "../_res",
+        "testmode": false
+    },
+    "name": "check cookies from server as defined in test request",
+    "tests": [
+        {
+            "name": "set a sess cookie",
+            "request": {
+                "server_url": "http://localhost:9999",
+                "endpoint": "bounce-json",
+                "method": "POST",
+                "x-test-set-cookie": [
+                    {
+                        "name": "sess",
+                        "value": "my_sess"
+                    },
+                    {
+                        "name": "dummy",
+                        "value": "whatever",
+                        "path": "/mypath",
+                        "domain": "mydomain",
+                        "expires": "2021-11-10T10:00:00Z",
+                        "max_age": 86400,
+                        "secure": false,
+                        "http_only": false,
+                        "same_site": 1
+                    }
+                ],
+                "body": {}
+            },
+            "response": {
+                "statuscode": 200,
+                "header": {
+                    "Set-Cookie": [
+                        "sess=my_sess",
+                        "dummy=whatever; Path=/mypath; Domain=mydomain; Expires=Wed, 10 Nov 2021 10:00:00 GMT; Max-Age=86400; SameSite"
+                    ]
+                },
+                "cookie": {
+                    "sess": {
+                        "name": "sess",
+                        "value": "mysess"
+                    },
+                    "dummy": {
+                        "name": "dummy",
+                        "value": "whatever",
+                        "path": "/mypath",
+                        "domain": "mydomain",
+                        "expires": "2021-11-10T10:00:00Z",
+                        "max_age": 86400,
+                        "secure": false,
+                        "http_only": false,
+                        "same_site": 1
+                    }
+                },
+                "body": {}
+            },
+			"store_response_qjson": {
+				"sess_cookie": "cookie.sess"
+			}
+        },
+        {
+            "name": "use cookie set in store",
+            "request": {
+                "server_url": "http://localhost:9999",
+                "endpoint": "bounce-json",
+                "method": "POST",
+                "cookies": {
+                    "sess": {
+                        "value_from_store": "sess_cookie"
+                    }
+                },
+                "body": {}
+            },
+            "response": {
+                "statuscode": 200,
+                "cookie": {
+                    "sess": {
+                        "name": "sess",
+                        "value": "mysess"
+                    }
+                },
```

Comment:
I don't really understand how the Cookie request to the test http server response with the same cookie, without having used the special "header-x-test-set-cookie"?